aks-node-controller/Makefile

```diff
@@ -0,0 +1,16 @@
+# Run buf in docker, mounting the full repo into the container
+# Emulate running "buf" in the current directory
+BUF = docker run --volume "$(CURDIR)/../:$(CURDIR)/../" --workdir $(CURDIR) bufbuild/buf:1.47.2
+
+.PHONY: proto-generate
```

Comment:
Do we need to manually run `make proto-generate` before pushing the commit?

Reply:
yes, CI may complain about it. But I didn't test it.